egs2/bean/cls1/README.md

```diff
@@ -0,0 +1,39 @@
+# Benchmark of Animal Sounds Recipe
+This recipe implements the audio classification task with a BEATs encoder and linear layer decoder model on the [The Benchmark of Animal Sounds (BEAN)](https://arxiv.org/abs/2210.12300).
+The data processing code is adapted from https://github.com/earthspecies/beans
+
+# Training Details and Requirements
+Most of the datasets complete within 5 hours and on a single L40 gpu.
+
+## Steps to run
```

Comment:
@popcornell This is a readme for the classification tasks in Beans (please ignore the name difference, I will correct that later). In addition to https://github.com/espnet/espnet/pull/6004/files#diff-c0c1c50802ecad0c14dfac23b3a8569edc27e6a234068a473fd0539e50d9f0a9
this should also be useful.

Reply:
Added via https://github.com/espnet/espnet/commit/e6e9a160f13282f393ae397811df37d210bbbc92 to base branch. Thanks!